app/views/providers/about_the_financial_assessments/show.html.erb

```diff
@@ -14,7 +14,7 @@
     </div>
     <div class="govuk-grid-column-one-quarter govuk-!-text-align-right">
       <%= govuk_link_to(
-            t("generic.change"),
+            t("generic.change_html", context: @applicant.email),
```

Comment:
You should be able to use `visually_hidden_suffix: @applicant.email` rather than adding your own custom context ([here's the documentation](https://govuk-components.netlify.app/helpers/link/#links-with-visually-hidden-text))

Also, out of interest, why did you choose the visually hidden text to say "Change [email address]" rather than "Change [section heading]"?

Reply:
@naseberry, recommended that I use i18n interpolation in the locales file and update the views to use the interpolation.

I used "Change [email address]" because clicking that "Change" button leads you to a page to change the client's email address.

Reply:
@skdejong But I mean it would say "Change xxx@example.com" rather than "Change your client's email address" (which is the section heading)

@naseberry Did you know you could do it my way?

Reply:
@agoldstone93  @skdejong  I did not know about `visually_hidden_suffix`, considering it is already built-into the helper, I would recommend using that instead.

Reply:
Thank you for your recommendations. I have made the recommended changes.